pkgs/applications/system/howdy/default.nix

```
@@ -0,0 +1,89 @@
+{ stdenv
+, lib
+, bzip2
+, fetchFromGitHub
+, fetchurl
+, fmt
+, gettext
+, inih
+, installShellFiles
+, libevdev
+, meson
+, ninja
+, pam
+, pkg-config
+, python3
+} @ args:
+let
+  data = import ./sources.nix args;
+in
+stdenv.mkDerivation {
+  pname = "howdy";
+  version = "unstable-2023-02-28";
+  inherit (data) src;
+
+  # fix paths
+  patches = [ ./howdy.patch ];
```

Please add an explanation, something like:

```suggestion
  patches = [
    # Change directory with configuration from `/etc` to `/var/lib`, since the service is expected to modify it.
    ./howdy.patch
  ];
```

Also keep the comment inside so the placement is not confusing when more patches are added.